doc/task-definition-example.yml

```diff
@@ -32,4 +32,15 @@ properties:
   - property_file: ../test/tasks/other.prp
     expected_verdict: true
 
+# Optional, contains dict with key-value pairs that
+# provide additional information about the context of the task definition.
+# If given, benchexec will pass this information to the tool.
+context:
+  # Machine model the task definition is defined for.
+  # Properties of this task definition are only guaranteed to be
+  # valid on the input_files for the machine model specified here.
+  #
+  # Values tools are expected to support: intel32 and intel64
+  - machine_model: intel32
```

Comment:
GCC and clang talk about `target`.
I don't like `architecture` because it is a very generic term:

software architecture, house architecture, target architecture, task architecture *(made that one up)*,